edge-middleware/feature-flag-hypertune/app/page.tsx

```diff
@@ -26,13 +27,16 @@ export default async function Home() {
           with Vercel Edge Config to initialize the Hypertune SDK with near-zero
           latency on the server so you can access your feature flags and run A/B
           tests with no performance impact to your app. It also shows how to
-          integrate with the Vercel Toolbar to let you easily set local flag
-          overrides while developing.
+          integrate with the Vercel Toolbar to let you easily view and override
+          your feature flags without leaving your frontend. Finally, it shows
+          how to use the Vercel Feature Flags pattern to use flags on a static
```

Comment:
To clarify a bit: we consider the ability to define flags in code as our feature flags pattern, and the ability to precompute some flags and use middleware to route between static pages is basically just an extension of that.

Reply:
We show the middleware + routing between static pages in this example so thought I'd call it out. But will update the wording here!